applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -3,7 +3,7 @@
 HID forward module
 ##################
 
-Use the HID forward module to:
```

Comment:
`Use the |hid_forward| to:`
Otherwise the word module will be added twice. Also in other places